agent/cmd/cuda-checkpoint-helper/main.cpp

```
@@ -0,0 +1,1599 @@
+  if (!output_capture.Start(&capture_setup_error) ||
+      !error_capture.Start(&capture_setup_error)) {
+    response.cuda_status = CUDA_ERROR_OPERATING_SYSTEM;
+    response.flags = daemon_protocol::kResponseFatal;
+    response.error = capture_setup_error;
+    return response;
+  }
+  (void)std::fflush(stdout);
+  (void)std::fflush(stderr);
+  const int saved_stdout = dup(STDOUT_FILENO);
+  const int saved_stderr = dup(STDERR_FILENO);
+  if (saved_stdout < 0 || saved_stderr < 0 ||
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)